specification/keyvault/resource-manager/Microsoft.KeyVault/preview/2021-04-01-preview/keys.json

```diff
@@ -0,0 +1,636 @@
+          "description": "The properties of the key."
+        }
+      },
+      "required": [
+        "properties"
+      ],
+      "allOf": [
+        {
+          "$ref": "./keyvault.json#/definitions/Resource"
+        }
+      ],
+      "description": "The key resource."
+    },
+    "KeyListResult": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Key"
+          },
+          "description": "The key resources."
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "The URL to get the next page of keys."
+        }
+      },
+      "description": "The page of keys."
+    },
+    "RotationPolicy": {
+      "properties": {
+        "attributes": {
+          "$ref": "#/definitions/KeyRotationPolicyAttributes",
+          "description": "The attributes of key rotation policy."
+        },
+        "lifetimeActions": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/LifetimeAction"
+          },
+          "description": "The lifetimeActions for key rotation action."
+        }
+      }
+    },
+    "KeyRotationPolicyAttributes": {
+      "properties": {
+        "created": {
+          "type": "integer",
+          "format": "int64",
+          "readOnly": true,
+          "description": "Creation time in seconds since 1970-01-01T00:00:00Z."
+        },
+        "updated": {
+          "type": "integer",
+          "format": "int64",
+          "readOnly": true,
+          "description": "Last updated time in seconds since 1970-01-01T00:00:00Z."
+        },
+        "expiryTime": {
+          "type": "string",
+          "description": "The expiration time for the new key version. It should be in ISO8601 format. Eg: 'P90D', 'P1Y'."
+        }
+      }
+    },
+    "LifetimeAction": {
+      "properties": {
+        "trigger": {
+          "$ref": "#/definitions/Trigger",
+          "description": "The trigger of key rotation policy lifetimeAction."
+        },
+        "action": {
+          "$ref": "#/definitions/Action",
+          "description": "The action of key rotation policy lifetimeAction."
+        }
+      }
+    },
+    "Trigger": {
+      "properties": {
+        "timeAfterCreate": {
+          "type": "string",
+          "description": "The time duration after key creation to rotate the key. It should be in ISO8601 format. Eg: 'P90D', 'P1Y'."
+        },
+        "timeBeforeExpiry": {
+          "type": "string",
+          "description": "The time duration before key expiring to rotate the key. It should be in ISO8601 format. Eg: 'P90D', 'P1Y'."
+        }
+      }
+    },
+    "Action": {
+      "properties": {
+        "type": {
+          "type": "string",
+          "description": "The type of action.",
+          "enum": [
+            "rotate",
+            "notify"
+          ],
+          "x-ms-enum": {
+            "name": "type",
```

Comment:
it conflicts with existing one. Please rename it

Reply:
Done